scripts/src/java/org/oppia/android/scripts/coverage/CoverageReporter.kt

```diff
@@ -2,27 +2,34 @@ package org.oppia.android.scripts.coverage
 
 import org.oppia.android.scripts.proto.Coverage
 import org.oppia.android.scripts.proto.CoverageReport
+import org.oppia.android.scripts.proto.CoverageReportContainer
+import org.oppia.android.scripts.proto.TestFileExemptions
 import java.io.File
 
+/** Minimum coverage percentage required. */
+const val MIN_THRESHOLD = 10 // to be decided
```

Comment:
Reminder to finalize this.

Reply:
Thanks for providing the min threshold, now its set to 70.